index.js

```diff
@@ -190,6 +190,23 @@ class HyperdriveFuse {
       })
     }
 
+    handlers.rename = function (oldpath, newpath, cb) {
+      log('rename', oldpath, newpath)
+      // This implementation only supports file rename, directories isn't really feasible for the moment
+      // TODO: Provide better renaming support once new trie rolls out
+      self.drive.stat(oldpath, (err,stat) =>{
+        if(stat.isFile()){
```

Comment:
Gotta handle the err here (the general pattern of `cb(-err.errno || Fuse.ENOENT)` should suffice).

Reply:
Also on the formatting side, just try to make sure the spacing/indentation (especially around your `{`s is consistent). I'm gonna enable the `standard` linter on this project as part of the `npm test` step to make this easier going forward.